Source/SwiftLintFramework/Rules/ValidDocsRule.swift

```diff
@@ -151,6 +151,12 @@ public struct ValidDocsRule: ConfigProviderRule {
             "\n/// - parameter param3: this is a void closure too" +
             "\nfunc a(param: () -> Void, param2: (parameter: Int) -> Void, " +
             "param3: (parameter: Int) -> Void) {}",
+            "/// docs\n/// - parameter param: this is a void closure" +
+            "\n/// - returns: Foo<Void>" +
```

Comment:
can you please indent multiline strings after the first line? It makes it easier to tell which strings belong together.